packages/bpk-component-datatable/package.json

```diff
@@ -19,7 +19,7 @@
     "bpk-react-utils": "^7.0.0",
     "lodash": "^4.17.20",
     "prop-types": "^15.7.2",
-    "react-virtualized": "^9.22.3"
+    "react-table": "7.8.0"
```

Comment:
Do we need to lock the react-table version or can we set the version to allow in range versions with `^`?